app/views/smart_listing/_action_custom.html.erb

```diff
@@ -0,0 +1,10 @@
+<%# 
```

Comment:
views/smart_listing is the default?
 
If yes, I think it isn't necessary generate these files, because the gem itself is already inclusive.